network.tf

```diff
@@ -0,0 +1,32 @@
+locals {
+  network_interface_names = ["nic1", "nic2", "nic3"]
+  security_rules = [
+    {
+      name                   = "allow-ssh"
+      priority               = 100
+      direction              = "Inbound"
+      access                 = "Allow"
+      protocol               = "Tcp"
+      source_port_range      = "*"
+      destination_port_range = "22"
+    },
+    {
+      name                   = "allow-http"
+      priority               = 200
+      direction              = "Inbound"
+      access                 = "Allow"
+      protocol               = "Tcp"
+      source_port_range      = "*"
+      destination_port_range = "80"
+    },
+    {
+      name                   = "allow-https"
+      priority               = 300
+      direction              = "Inbound"
+      access                 = "Allow"
+      protocol               = "Tcp"
+      source_port_range      = "*"
+      destination_port_range = "443"
+    }
+  ]
+}
```

Comment:
This block is identical to the one in `locals.tf`. Consider removing the duplicate or refactoring your code to avoid redundancy.